gemcalibration/html/scripts/shelfMonitor.js

```diff
@@ -0,0 +1,92 @@
+var actionURL = "";
```

Comment:
Please remove this file from the project. You're not using it and will have to write your own script later.